src/features/dotnetTest.ts

```diff
@@ -481,19 +441,19 @@ class DebugEventListener {
                     event = DebuggerEventsProtocol.decodePacket(buffer);
                 }
                 catch (e) {
-                    this._outputChannel.appendLine("Warning: Invalid event received from debugger");
+                    this._eventStream.post(new DebuggerWarning("Invalid event received from debugger"));
```

Comment:
Let's rename this event to mention that it part of the DotNetTest feature and not the general debugger.